pkg/http/server_test.go

```diff
@@ -97,3 +97,66 @@ func TestNewServer(t *testing.T) {
 		}
 	}
 }
+
+var tlsCipherSuitesTests = []struct {
+	ciphers    []uint16
+	shouldFail bool
+}{
+	{ciphers: nil, shouldFail: false},
```

Comment:
You don't need to specify fields here.  You could directly give values like `{nil, false}`

Reply:
Sure, but it improves readability - Makes things explicit

Reply:
I don't agree for this case, repeated `ciphers` and `shouldFail` do not look nice.  But the call is yours.

Reply:
Here it makes no big difference but e.g. the erasure_read tests would be very hard to read without explicit names: https://github.com/minio/minio/blob/master/cmd/erasure-readfile_test.go#L42